src/scss/components/global.scss

```diff
@@ -11,3 +11,14 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
```

Comment:
https://styled-components.com/docs/basics#extending-styles - popular library for CSS customizing - optional